pkg/controllers/proxy/network_services_controller.go

```diff
@@ -735,75 +744,154 @@ func (nsc *NetworkServicesController) publishMetrics(serviceInfoMap serviceInfoM
 		}
 	}()
 
-	ipvsSvcs, err := nsc.ln.ipvsGetServices()
+	ipvsHandle, err := ipvs.New("")
```

Comment:
I'm not sure that I understand why you replaced `nsc.ln.ipvsGetServices()` with a manual call to `ipvsHandle.GetServices()`. Can you explain this a bit more?

Reply:
Thanks for the clarification.